lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -44,7 +44,7 @@ public class Timeout : Name<int>
         {
         }
 
-        [NamedParameter("Retry times", defaultValue: "5")]
+        [NamedParameter("Retry times", defaultValue: "500")]
         public class RetryCount : Name<int>
```

Comment:
Just setting it to an arbitrary number that happens to work seems dangerous. We should at least add the above reasoning to the comments of the code to make sure we remember why we choose 500 as the magic number.